### ISSUE TITLE
Dual package with native ES Modules output

### ISSUE THREAD

**Issue by dandv** [sindresorhus/serialize-error#36]
Thanks for this super useful package!

Any chance to publish a [dual package](https://alexomara.com/blog/dual-packages-in-node-js-conditional-exports/) with support for native `.mjs` imports?

**Comment by sindresorhus** [comment]
No. I plan to switch over completely to ESM all at once, at some point.
